### PR TITLE
CORE-9 Add Swagger docs to the filesystem stat endpoint.

### PR DESCRIPTION
**Pull request by psarando**
This PR will add the schemas and docs added by cyverse-de/common-swagger-api#52 to the  `filesystem/stat` endpoint.

The `terrain.routes` namespace was also reformatted with `lein-cljfmt`

### CODE REVIEW

File: src/terrain/routes.clj [hunk 1]
```diff
@@ -95,57 +95,58 @@
...
+   (filesystem-stat-routes)
+   (filesystem-ticket-routes)
+   (secured-filesystem-routes)
+   (secured-filesystem-metadata-routes)
+   (secured-search-routes)
+   (secured-oauth-routes)
+   (secured-favorites-routes)
+   (secured-tag-routes)
+   (data-comment-routes)
+   (route/not-found (service/unrecognized-path-response))))
```
Comment (slr71) [inline src/terrain/routes.clj]: 👍

File: src/terrain/routes.clj [hunk 2]
```diff
@@ -95,57 +95,58 @@
...
+   (admin-ontology-routes)
+   (admin-reference-genomes-routes)
+   (admin-tool-routes)
+   (admin-tool-request-routes)
+   (admin-permanent-id-request-routes)
+   (oauth-admin-routes)
+   (admin-integration-data-routes)
+   (admin-workspace-routes)
+   (admin-user-info-routes)
+   (route/not-found (service/unrecognized-path-response))))
```
Comment (slr71) [inline src/terrain/routes.clj]: 👍

File: src/terrain/services/filesystem/stat.clj [hunk 3]
```diff
@@ -97,39 +95,26 @@
...
-    (validate-map body {:paths vector?})
-    (validate-map body {:paths #(not (empty? %1))})
-    (validate-map body {:paths #(every? (comp not string/blank?) %1)})
-    (validators/validate-num-paths (:paths body))))
-
-(with-post-hook! #'do-stat (paths/log-func "do-stat"))
+  [{user :user} body]
+  (-> (data-raw/collect-stats user body)
+      :body
+      (json/decode true)))
```
Comment (slr71) [inline src/terrain/services/filesystem/stat.clj]: 👍

**Comment by psarando** [comment]
Thanks for the review! 👍

**Pull request merged**